wp-content/themes/core/components/navigation/Navigation_Controller.php

```diff
@@ -8,75 +8,122 @@
 
 class Navigation_Controller extends Abstract_Controller {
 
-	public const LOCATION         = 'location';
-	public const CLASSES          = 'classes';
 	public const ATTRS            = 'attrs';
+	public const CLASSES          = 'classes';
+	public const MENU             = 'menu';
+	public const MENU_LOCATION    = 'menu_location';
+	public const NAV_LIST_ATTRS   = 'nav_list_attrs';
 	public const NAV_LIST_CLASSES = 'nav_list_classes';
 	public const NAV_MENU_ARGS    = 'nav_menu_args';
 
 	private const DEFAULT_NAV_MENU_ARGS = [
-		'container'   => false,
-		'menu_class'  => '',
-		'menu_id'     => '',
-		'depth'       => 1,
-		'items_wrap'  => '%3$s',
-		'fallback_cb' => false,
+		'menu'           => null,
+		'container'      => false,
+		'menu_class'     => '',
+		'menu_id'        => '',
+		'depth'          => 1,
+		'items_wrap'     => '%3$s',
+		'fallback_cb'    => false,
+		'theme_location' => '',
 	];
 
-	private string $location;
-	private array $classes;
+	/**
+	 * @var array
```

Comment:
🟠 
Should fix: Since we are declaring the types, we don't need the var comments anymore.